[2025-01-31T11:25:06.965+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-01-31T11:25:07.013+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: sales-analysis-dag.save_transformed_file manual__2025-01-31T11:24:24.409295+00:00 [queued]>
[2025-01-31T11:25:07.038+0000] {taskinstance.py:2613} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: sales-analysis-dag.save_transformed_file manual__2025-01-31T11:24:24.409295+00:00 [queued]>
[2025-01-31T11:25:07.040+0000] {taskinstance.py:2866} INFO - Starting attempt 1 of 2
[2025-01-31T11:25:07.084+0000] {taskinstance.py:2889} INFO - Executing <Task(PythonOperator): save_transformed_file> on 2025-01-31 11:24:24.409295+00:00
[2025-01-31T11:25:07.100+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=936) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-01-31T11:25:07.105+0000] {standard_task_runner.py:72} INFO - Started process 938 to run task
[2025-01-31T11:25:07.106+0000] {standard_task_runner.py:104} INFO - Running: ['***', 'tasks', 'run', 'sales-analysis-dag', 'save_transformed_file', 'manual__2025-01-31T11:24:24.409295+00:00', '--job-id', '33', '--raw', '--subdir', 'DAGS_FOLDER/sales_analysis.py', '--cfg-path', '/tmp/tmp4471_5u2']
[2025-01-31T11:25:07.109+0000] {standard_task_runner.py:105} INFO - Job 33: Subtask save_transformed_file
[2025-01-31T11:25:07.842+0000] {task_command.py:467} INFO - Running <TaskInstance: sales-analysis-dag.save_transformed_file manual__2025-01-31T11:24:24.409295+00:00 [running]> on host df376234e961
[2025-01-31T11:25:08.232+0000] {taskinstance.py:3132} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='olalekanrasaq1331@gmail.com' AIRFLOW_CTX_DAG_OWNER='Akinkunmi' AIRFLOW_CTX_DAG_ID='sales-analysis-dag' AIRFLOW_CTX_TASK_ID='save_transformed_file' AIRFLOW_CTX_EXECUTION_DATE='2025-01-31T11:24:24.409295+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-01-31T11:24:24.409295+00:00'
[2025-01-31T11:25:08.236+0000] {logging_mixin.py:190} INFO - Task instance is in running state
[2025-01-31T11:25:08.237+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: queued
[2025-01-31T11:25:08.238+0000] {logging_mixin.py:190} INFO - Current task name:save_transformed_file state:running start_date:2025-01-31 11:25:07.015326+00:00
[2025-01-31T11:25:08.240+0000] {logging_mixin.py:190} INFO - Dag name:sales-analysis-dag and current dag run status:running
[2025-01-31T11:25:08.241+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-01-31T11:25:10.885+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-01-31T11:25:10.933+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-01-31T11:25:10.936+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=sales-analysis-dag, task_id=save_transformed_file, run_id=manual__2025-01-31T11:24:24.409295+00:00, execution_date=20250131T112424, start_date=20250131T112507, end_date=20250131T112510
[2025-01-31T11:25:10.992+0000] {logging_mixin.py:190} INFO - Task instance in success state
[2025-01-31T11:25:10.994+0000] {logging_mixin.py:190} INFO -  Previous state of the Task instance: running
[2025-01-31T11:25:10.995+0000] {logging_mixin.py:190} INFO - Dag name:sales-analysis-dag queued_at:2025-01-31 11:24:24.506798+00:00
[2025-01-31T11:25:10.998+0000] {logging_mixin.py:190} INFO - Task hostname:df376234e961 operator:PythonOperator
[2025-01-31T11:25:11.103+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-01-31T11:25:11.494+0000] {taskinstance.py:3895} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-01-31T11:25:11.504+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
[2025-02-03T05:46:57.540+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-02-03T05:46:57.584+0000] {taskinstance.py:2603} INFO - Dependencies not met for <TaskInstance: sales-analysis-dag.save_transformed_file manual__2025-01-31T11:24:24.409295+00:00 [success]>, dependency 'Task Instance State' FAILED: Task is in the 'success' state.
[2025-02-03T05:46:57.654+0000] {local_task_job_runner.py:166} INFO - Task is not able to be run
